winui/Cartesian-Charts/Appearance.md

```diff
@@ -47,117 +32,43 @@ Each palette applies a set of predefined brushes to the series in a predefined o
+[SfCartesianChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCartesianChart.html) provides support to define own brushes for the chart with preferred order by using the [PaletteBrushes]() property, as shown in the following code example.
 
-{% tabs %}
+## Applying PaletteBrushes for Series
 
-{% highlight xaml %}
-
-<chart:ColumnSeries ItemsSource="{Binding Data}"  
-                    XBindingPath="Demand" 
-                    YBindingPath="Year2010" 
-                    Palette="GreenChrome"/>
-
-{% endhighlight %}
-
-{% highlight c# %}
-
-ColumnSeries series1 = new ColumnSeries()
-{
-    ItemsSource = new ViewModel().Data,
-    XBindingPath = "Demand",
-    YBindingPath = "Year2010",
-    Palette = ChartColorPalette.GreenChrome
-};
-
-{% endhighlight %}
-
-{% endtabs %}
-
-![Series palette support in WinUI Chart](Appearance_images/WinUI_chart_greenchrome_palette.png)
-
-## Custom Palette
-
-[SfCartesianChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCartesianChart.html) provides support to define own brushes for the chart with preferred order by using the [CustomBrushes](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartColorModel.html#Syncfusion_UI_Xaml_Charts_ChartColorModel_CustomBrushes) property of [ChartColorModel](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartColorModel.html) and [Palette](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartBase.html#Syncfusion_UI_Xaml_Charts_ChartBase_Palette) value as `Custom`, as shown in the following code example.
+Cartesian chart provides support to set the palette to series for applying predefined brushes to the segment. The following code example shows you how to set the [PaletteBrushes]() for the series.
 
 {% tabs %}
 
-{% highlight xaml %}
-
-<chart:SfCartesianChart.Series>
-    <chart:ColumnSeries ItemsSource="{Binding Data}"  
-                        XBindingPath="Demand" 
-                        YBindingPath="Year2010" 
-                        Palette="Custom">
-        <chart:ColumnSeries.ColorModel>
-            <chart:ChartColorModel>
-                <chart:ChartColorModel.CustomBrushes>
-                    <SolidColorBrush Color="#26c6da"/>
-                    <SolidColorBrush Color="#00bcd4"/>
-                    <SolidColorBrush Color="#00acc1"/>
-                    <SolidColorBrush Color="#0097a7"/>
-                    <SolidColorBrush Color="#00838f"/>
-                </chart:ChartColorModel.CustomBrushes>
-            </chart:ChartColorModel>
-        </chart:ColumnSeries.ColorModel>
-    </chart:ColumnSeries>
-</chart:SfCartesianChart.Series>
-
-{% endhighlight %}
-
```

Comment:
@YuvarajPalanisamy : Why you have removed the xaml section

Reply:
I have added the Bruch colllection in Xaml code example